src/SFA.DAS.FAA.Application/Queries/GetSavedVacancies/GetSavedVacanciesQuery.cs

```diff
@@ -54,7 +64,22 @@ public class GetSavedVacanciesQueryHandler(IApiClient apiClient) : IRequestHandl
         public async Task<GetSavedVacanciesQueryResult> Handle(GetSavedVacanciesQuery request, CancellationToken cancellationToken)
         {
             var response = await apiClient.Get<GetSavedVacanciesApiResponse>(
-                new GetSavedVacanciesApiRequest(request.CandidateId));
+                new GetSavedVacanciesApiRequest(request.CandidateId)) ?? new GetSavedVacanciesQueryResult();
+
+            if (string.IsNullOrEmpty(request.DeleteVacancyReference)) return response;
+
+            if (!request.DeleteVacancyReference.StartsWith("VAC", StringComparison.CurrentCultureIgnoreCase))
+            {
+                request.DeleteVacancyReference = $"VAC{request.DeleteVacancyReference}";
```

Comment:
Same as above - we should allow for the VAC being sent in the request

Reply:
I have removed it from the web.